server/src/connectionHandlers/onDefinition.ts

```diff
@@ -104,6 +104,28 @@ export function onDefinitionHandler (textDocumentPositionParams: TextDocumentPos
       })
       return definitions
     }
+    // Overrides
+    if (analyzer.isOverride(documentUri, position.line, position.character)) {
```

Comment:
This function is huge and easily split into smaller ones for each kind of symbol. Consider doing a refactoring in a separate PR?

Reply:
Oh right, if we haven't done the recipe specific scan, then we get the global scan results which is nice. In that case, you could move recipe scan before global scan since it's more accurate.

Reply:
In that case, you can move the recipe scan first because it's more accurate

Reply:
done. af6cb9ad185f9f44d219666bface48d3ee03ba52